src/core/web_api/p2_api.js

```diff
@@ -34,11 +34,16 @@ class Auth0APIClient {
       _telemetryInfo: opts._telemetryInfo || default_telemetry
     });
 
+    var state = opts.state;
+    if (opts.params && opts.params.state) {
+      state = opts.params.state;
```

Comment:
@luisrudge - if they have both then the former example in your description will override the latter. Is that expected? Is it consistently documented one way or the other?